php/class-media.php

```diff
@@ -671,6 +672,17 @@ public function get_public_id_from_url( $url, $as_sync_key = false ) {
 		// Is SEO friendly URL.
 		if ( in_array( 'images', $maybe_seo, true ) ) {
 			$public_id = $path_info['dirname'];
+		} elseif ( false !== strpos( $url, '/image/fetch/' ) ) {
+			// Maybe the $file is already the URL - $url has the version.
+			if ( filter_var( $file, FILTER_VALIDATE_URL ) ) {
```

Comment:
@pereirinha perhaps utf8 encode the url in case of international chars?

Reply:
@DavidCramer, please check the comment about this is the backlog.